sdk/include/reactos/wine/test.h

```diff
@@ -634,30 +634,24 @@ const char *wine_dbgstr_an( const CHAR *str, intptr_t n )
     return res;
 }
 
-const char *wine_dbgstr_wn( const WCHAR *str, intptr_t n )
+const char *wine_dbgstr_wn( const WCHAR *str, int n )
```

Comment:
No idea what Wine drank, but this should certainly be `intptr_t` or `size_t` for x64 compliance.

Reply:
Changed. Thanks.